data/presets/amenity/townhall/barangay.json

```
@@ -0,0 +1,25 @@
+{
+    "locationSet": {
+        "include": [
+            "ph"
+        ]
+    },
+    "icon": "fas-building-flag",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "barangay hall"
+    ],
+    "tags": {
+        "amenity": "townhall",
+        "townhall:type": "barangay",
+        "admin_level" : "10"
+    },
+    "reference": {
+        "key": "townhall:type",
+        "value": "barangay"
+    },
```

as far as I can see, there is no wiki page dedicated to the `townhall:type=barnangay` tag yet (see https://wiki.openstreetmap.org/wiki/Tag:townhall:type=barangay).

We could either link to the documentation page of the tag key (https://wiki.openstreetmap.org/wiki/Key:townhall:type) or `amentiy=townhall` (https://wiki.openstreetmap.org/wiki/Tag:amenity=townhall) instead. I'd slightly lean towards the latter:

```suggestion
    "reference": {
        "key": "amenity",
        "value": "townhall"
    },
```

---

That sounds reasonable, and thanks for bringing up the missing wiki page. I should write a stub to get it started. Thanks!